### ISSUE TITLE
Region specific permission

### ISSUE THREAD

**Issue by Magi1053** [JimiIT92/UniverseGuard#10]
Is there any functionality to be able to give a specific permission node the ability to override the rules set for that region? For instance I defined a region around spawn but I want to give selective people access to build there without giving them access to the whole plugin.

**Comment by JimiIT92** [comment]
Yes, the plugin has 2 permissions: one to give access to all commands and stuffs, and one just to allow user to bypass region permissions. So if you give this second permission to someone they will be able to do whatever they want in region but they can't create new ones or delete existing ones

**Comment by Magi1053** [comment]
the permission to bypass will bypass all regions.. I only want to be able to bypass one specific region

**Comment by JimiIT92** [comment]
Ah, now i understand. In this case i will add a way to define specific permissions for the single region or to specify who can bypass region restrictions ;)

**Comment by Nbgreen** [comment]
any updates on this?

**Comment by JimiIT92** [comment]
This will be implemented in Version 3, wich is work in progress. However due to very limited spare time due to work is taking waaaaaay longer than expected to finish it :(
